test/scenarios/verifier/.pre-commit-config.yaml

```diff
@@ -0,0 +1,36 @@
+---
+repos:
+  - repo: git://github.com/pre-commit/pre-commit-hooks
+    rev: v2.0.0
+    hooks:
+      - id: fix-encoding-pragma
+      - id: trailing-whitespace
+      - id: flake8
+        args: [--max-line-length=120]
+      - id: check-merge-conflict
+      - id: double-quote-string-fixer
+      - id: end-of-file-fixer
+      - id: debug-statements
+      - id: check-added-large-files
+      - id: check-ast
+      - id: check-byte-order-marker
+      - id: check-case-conflict
+      - id: check-docstring-first
+      - id: check-executables-have-shebangs
+        types: [python]
+      - id: check-vcs-permalinks
+      - id: mixed-line-ending
+      - id: check-symlinks
+      - id: detect-private-key
+  - repo: local
+    hooks:
+      - id: pylint
+        name: pylint
+        entry: python -m pylint
+        language: system
+        types: [python]
+  - repo: git://github.com/Lucas-C/pre-commit-hooks
```

Comment:
Can we avoid an external repository? Unsure if all these checks are related to that but can we have instead a 'hello, world' check or something that just smokes out that it runs anything successfully. Sorry for my continued lack of pre-commit knowledge :)

Reply:
Is this related to https://github.com/ansible/molecule/issues/1788#issuecomment-468806847?

Reply:
Some of it is just my own communication with you guys, to show you the extent of pre-commit and what it can do. I agree that for the fixture a local hook that just runs something simple would be preferred. However, I would think that a pre-commit config like this could easily take the place of the lint and format checks in your `tox.ini` at the root of the repository.

Reply:
I'll just run the flake8 hook since that's what you guys run through tox